assets/tests/components/alertsPage.test.tsx

```diff
@@ -32,14 +32,21 @@ describe("Alerts Page", () => {
 
   describe("filtering", () => {
     test("filters places by mode and route", async () => {
-      const { getByRole, findByRole, getByTestId } = render(
+      const { getByRole, findByRole, getByTestId, queryByTestId } = render(
         <AlertsPage
           places={placesAndScreens as Place[]}
           screensByAlertId={alertsOnScreens as ScreensByAlert}
           isVisible={true}
         />
       );
 
+      // First, verify expired alerts are not visible
+      // It's possible this filtering will get done in Christian's branch; check
+      // expect(queryByTestId("9")).not.toBeInTheDocument();
```

Comment:
I ran the test after uncommenting this and it passes. So you should be good on uncommenting and removing comments.